test/benchmarks/Benchmarks.Trace/AgentWriterBenchmark.cs

```diff
@@ -41,11 +44,20 @@ static AgentWriterBenchmark()
                 EnrichedSpans[i] = new Span(new SpanContext((ulong)i, (ulong)i, SamplingPriority.UserReject, "Benchmark", null), now);
                 EnrichedSpans[i].SetTag(Tags.Env, "Benchmark");
                 EnrichedSpans[i].SetMetric(Metrics.SamplingRuleDecision, 1.0);
+                //
+                SpansWithOrigin[i] = new Span(new SpanContext((ulong)i, (ulong)i, SamplingPriority.UserReject, "Benchmark", null), now);
+                SpansWithOrigin[i].SetTag(Tags.Origin, "synthetics");
+                EnrichedSpansWithOrigin[i] = new Span(new SpanContext((ulong)i, (ulong)i, SamplingPriority.UserReject, "Benchmark", null), now);
+                SpansWithOrigin[i].SetTag(Tags.Origin, "synthetics");
```

Comment:
```suggestion
                EnrichedSpansWithOrigin[i].SetTag(Tags.Origin, "synthetics");
```
Is this what you meant to do?

Reply:
true!, my bad. Thanks!